include/glow/Backends/CompiledFunction.h

```diff
@@ -16,12 +16,47 @@
 #ifndef GLOW_BACKENDS_COMPILEDFUNCTION_H
 #define GLOW_BACKENDS_COMPILEDFUNCTION_H
 
+#include "glow/Graph/Nodes.h"
 #include <unordered_map>
 
 namespace glow {
 
 class Context;
+namespace runtime {
+/// Runtime Symbol Info
+/// Contains Information for initialization and handling of symbol at runtime.
 
+struct RuntimeSymbolInfo {
+  /// \p size is the size in bytes of the symbol.
+  size_t size;
+
+  /// \p offset is the offset in bytes from the base address.
+  size_t offset;
```

Comment:
More a remark than anything. With an offset we assume that the runtime has easily access to the base address and that it knows easily how to map a symbol to a base address.
For instance, let say I ask where is `foo`, we need to know somewhere that `foo` is a weight or activation to look at the right table.
More generally, again just future looking remarks, if we start to share weights between different graphs (as in not different instances of the same graph, but really different graphs[1]), we may want to create more fine tuned pool of symbols and we would need to somehow keep track of which symbol belongs to which pool.
The bottom line is moving forward, I wonder if this should be a full address instead of just an offset (or a base + offset in case we want to support relocating things)

[1] This could easily happen if we were to split a graph in the middle. E.g.,
Split
```
Weights
  |   \
  V    \
 op     \
  |     |
  V     |
 cpu Op |
  \     |
   \   /
    V V
    op
```
Into
```
Weights   [shared between G1 & G3]
--|---\---- vv G1 on accelerator A
  V    \
 op     \
--|-----|--- vv G2 on CPU
  V     |
 cpu Op |
  \     |
---\---/--- vv G3 on accelerator A
    V V
    op
```

Reply:
Summarizing the conversations between @opti-mix  and @qcolombet. I think adding a "kind" value to symbolInfo that allows interchange between offset and absolute memory addressing is a great idea, but not needed for this PR. We will revisit when we looks at expanding the runtime API perhaps in #1951 .